libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Message.cs

```diff
@@ -31,6 +29,6 @@ public class Message
         /// Gets or sets the quick replies.
         /// </summary>
         /// <value>The quick replies array.</value>
-        public List<object> QuickReplies { get; set; }
+        public List<object> QuickReplies { get; set; } = new List<object>();
```

Comment:
Is the set here necessary? If not, remove it and leave just the new List<().

Reply:
It is necessary. It's used in FacebookAdapter